app/views/admins/surveys/_form.html.erb

```diff
@@ -23,6 +23,20 @@
       </div>
     </div>
 
+    <div>
+      <%= render Forms::TextFieldComponent.new(form: f, method: :start_date, type: 'datetime-local', value: @survey.start_date&.strftime('%Y-%m-%dT%H:%M')) %>
```

Comment:
not now, but I think it would be worth DRYing this up into a subclass of TextFieldComponent specifically for datetime fields.

Reply:
Great, I will add it